docs/README.md

```diff
@@ -1044,6 +1044,133 @@ docker-compose up
 2. Site will be avaiable on `localhost:3001`
 3. You can update the documentation by modifying the file `docs/README.md`
 
+## OpReturn
+
+OP_RETURN is a [script](https://wiki.bitcoinsv.io/index.php/OP_RETURN) opcode that allows adding extra information to transactions beyond standard inputs and outputs. This can be useful for applications that want to include:
+
+* Different transaction types related to events (e.g., gaming);
+
+* Invoice numbers;
+
+* Nonces to differentiate payments of the same amount;
+
+* Other use cases.
+
+### General Syntax Rules:
+
+OpReturn is composed by three mandatory components:
+
+1.  **Version**: A byte allowing future updates (currently always `0`).
+
+2.  **Data Payload**: Custom information in UTF-8 format with the indicated size (can be empty).
+
+3.  **Nonce**: Eight random bytes to differentiate payments (can be empty).
+
+If the **data payload** or **nonce** is empty, it is represented by the `OP_0` [code](https://wiki.bitcoinsv.io/index.php/Opcodes_used_in_Bitcoin_Script).
+
+
+### General OpReturn structure
+
+    <OP_RETURN hex> <pushdata for protocol identifier> <protocol identifier> <version> <data> <nonce>
```

Comment:
pushdata for data and nonce should also have been mentioned

Reply:
This could be before the syntax rules explanation, it's would be more logical to see the structure before an explanation of the components.